components/statuscake/sources/new-uptime-alert/new-uptime-alert.mjs

```diff
@@ -0,0 +1,68 @@
+import statuscake from "../../statuscake.app.mjs";
+
+export default {
+  name: "New Uptime Alert",
+  version: "0.0.1",
+  key: "statuscake-new-uptime-alert",
+  description: "Emit new event on each new uptime alert.",
+  type: "source",
+  dedupe: "unique",
+  props: {
+    statuscake,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      static: {
+        intervalSeconds: 15 * 60, // 15 minutes
+      },
+    },
+    uptimeId: {
+      propDefinition: [
+        statuscake,
+        "uptimeId",
+      ],
+    },
+  },
+  methods: {
+    emitEvent(data) {
+      this.$emit(data, {
+        id: data.id,
+        summary: `New alert with id ${data.id}`,
+        ts: new Date(),
+      });
+    },
+  },
+  hooks: {
+    async deploy() {
+      const alerts = await this.statuscake.getAlerts({
+        uptimeId: this.uptimeId,
+        params: {
+          limit: 10,
+        },
+      });
+
+      alerts.reverse().forEach(this.emitEvent);
+    },
+  },
+  async run() {
```

Comment:
If it is possible(e.g. if new IDs are always greater than older ones), can we store latest IDs(or some other info) to compare and differentiate new alerts? Because it seems that when timer ticks, all alerts will be fetched.